spec/models/user_spec.rb

```diff
@@ -1495,6 +1495,55 @@ def it_should_not_send_survey
     end
   end
 
+  describe '#new_device' do
```

Comment:
This fell out of sync when it was renamed:

```suggestion
  describe '#new_device?' do
```